entity-types/ext-triton/summary_metrics.yml

```diff
@@ -0,0 +1,34 @@
+nv_cpu_memory_total_bytes:
+  goldenMetric: nv_cpu_memory_total_bytes
```

Comment:
You need to reflect the new golden metric names from the other file

Reply:
Hi @lujop,
As per your suggestions, I have made changes. Could you please review it.

Reply:
goldenMetric name doesn't reflect changed name

Reply:
yes, updated.